sql/src/main/java/io/crate/metadata/doc/CrateMetaDataUpgradeService.java

```diff
@@ -100,7 +99,6 @@ private IndexTemplateMetaData upgradeTemplateMapping(IndexTemplateMetaData index
             indexTemplateMetaData.getSettings().get(IndexMetaData.SETTING_LEGACY_ROUTING_HASH_FUNCTION));
         if (mappingMetaData != newMappingMetaData) {
             logger.info("upgraded mapping of template={}", indexTemplateMetaData.getName());
-            markAsUpgraded(newMappingMetaData);
```

Comment:
why that change? was here by intend, maybe we will have different upgrade reasons/methods to check in future